rosplane/scripts/rosplane_gcs_launch.sh

```diff
@@ -0,0 +1,196 @@
+
+tmux source-file ~/.tmux.conf
+
+# Split the tmux window into 4 panes
+tmux split-window -t rosplane_sim_session:0.0 -h
+tmux split-window -t rosplane_sim_session:0.0 -v
+tmux split-window -t rosplane_sim_session:0.2 -v
+
+# Arrange panes in each corner
+tmux select-pane -t rosplane_sim_session:0.0
+tmux select-pane -t rosplane_sim_session:0.1
+tmux select-pane -t rosplane_sim_session:0.2
+tmux select-pane -t rosplane_sim_session:0.3
+
+# Window placement reference:
+# rosplane_sim_session:0.0 Top Left
+# rosplane_sim_session:0.1 Bottom Left
+# rosplane_sim_session:0.2 Top Right
+# rosplane_sim_session:0.3 Bottom Right
+
+# Setup all panes to ssh into the remote
+if $online; then
+  tmux send-keys -t rosplane_sim_session:0.0 "ssh $user_host" C-m
+  tmux send-keys -t rosplane_sim_session:0.1 "ssh $user_host" C-m
+  tmux send-keys -t rosplane_sim_session:0.2 "ssh $user_host" C-m
+  tmux send-keys -t rosplane_sim_session:0.3 "ssh $user_host" C-m
+fi 
+
+# Send all of the panes to the working directory.
+tmux send-keys -t rosplane_sim_session:0.0 "cd $filepath" C-m
+tmux send-keys -t rosplane_sim_session:0.1 "cd $filepath" C-m
+tmux send-keys -t rosplane_sim_session:0.2 "cd $filepath" C-m
+tmux send-keys -t rosplane_sim_session:0.3 "cd $filepath" C-m
+
+# TODO test docker implementation.
+
+if $docker; then
+  tmux send-keys -t rosplane_sim_session:0.0 "docker compose up -d" C-m
+  sleep 2
+  tmux send-keys -t rosplane_sim_session:0.0 "docker compose exec $container bash" C-m # TODO check these commands.
+  tmux send-keys -t rosplane_sim_session:0.1 "docker compose exec $container bash" C-m
+  tmux send-keys -t rosplane_sim_session:0.2 "docker compose exec $container bash" C-m
+  tmux send-keys -t rosplane_sim_session:0.3 "docker compose exec $container bash" C-m
+fi
+
+# Send commands to run the sim.
+
+if $sim; then
+  tmux send-keys -t rosplane_sim_session:0.0 "ros2 launch rosflight_sim fixedwing_sim_io_joy.launch.py aircraft:=$aircraft" C-m
+else
+  tmux send-keys -t rosplane_sim_session:0.0 "ros2 run rosflight_io rosflight_io --ros-args -p port:=/dev/ttyACM0" C-m
+fi
+
+if $tuning; then
+  if $sim; then
+    tmux send-keys -t rosplane_sim_session:0.2 "ros2 launch rosplane_sim sim_tuning.launch.py aircraft:=$aircraft" C-m
+  else 
+    tmux send-keys -t rosplane_sim_session:0.2 "ros2 launch rosplane_tuning rosplane_tuning.launch.py aircraft:=$aircraft" C-m
+  fi 
+elif $sim; then
+  tmux send-keys -t rosplane_sim_session:0.2 "ros2 launch rosplane_sim sim.launch.py aircraft:=$aircraft" C-m
+else
+  tmux send-keys -t rosplane_sim_session:0.2 "ros2 launch rosplane rosplane.launch.py aircraft:=$aircraft" C-m
+fi
+
+
+# Check to see if rc_sim has been passed an arg
+if $rc_sim; then
+  tmux send-keys -t rosplane_sim_session:0.3 'ros2 run rosflight_sim rc_sim.py --ros-args --remap RC:=/fixedwing/RC' C-m
+fi
+
+if $sim; then
+  sleep 3 # It doesn't look like there is a clear way to do this better. This will have to do.
```

Comment:
I suppose we could add a ros2 service call that performs a delayed calibration. Not really necessary though, 3 seconds isn't too bad.

Reply:
Or modify the bias equation to save/load their biases to disk, so they are consistent between restarts. This is an issue for multirotors too, so it may be worth looking into.

Reply:
I think that we should look into a seeded random bias.

Reply:
I am going to leave this as is, and instead look into a seeded bias for rosflight_sim.